ui/components/input/base/_index.scss

```diff
@@ -107,7 +107,7 @@
   }
   .nds-form-element__label {
     color: $color-text-input-disabled;
-    opacity: 0.5;
+    //opacity: 0.5;  /** Commenting as we should not reduce the color contrast for readonly */
```

Comment:
Please check with Krishna or Rampi if this is fine, since it will change the way customer sees read-only fields

Reply:
@avnesh-neev Confirmed that we should not change the Opacity in Read only mode and same behaviour is there in SLDS. We observed that having the opacity is reducing the colour contrast and triggering the accessibility issue. I will discuss with Krishna as well and update.

Reply:
@rahulnsf FYI - Confirmed with Krishna.